store/CHANGELOG.md

```diff
@@ -53,3 +53,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 * [#14746](https://github.com/cosmos/cosmos-sdk/pull/14746) The `store` module is extracted to have a separate go.mod file which allows it be a standalone module.
 * [#14410](https://github.com/cosmos/cosmos-sdk/pull/14410) `rootmulti.Store.loadVersion` has validation to check if all the module stores' height is correct, it will error if any module store has incorrect height.
+
+### Improvements
+
+* [#17158](https://github.com/cosmos/cosmos-sdk/pull/17158) Start the goroutine after need to create a snapshot.
```

Comment:
Could you move the changelog under unreleased?

Reply:
OK